docs/build/apps/dapp-frontend.mdx

```diff
@@ -19,25 +19,32 @@ Let's get started.
 
 You're going to need [Node.js](https://nodejs.org/en/download/package-manager/) v18.14.1 or greater. If you haven't yet, install it now.
 
-We want to initialize our current project as an Astro project. To do this, we can again turn to the `stellar contract init` command, which has a `--frontend-template` flag that allows us to pass the url of a frontend template repository. As we learned in [Storing Data](../smart-contracts/getting-started/storing-data.mdx#adding-the-increment-contract), `stellar contract init` will not overwrite existing files, and is safe to use to add to an existing project.
+We want to initialize our current project as an Astro project. To do this, we can clone a template. You can find Soroban templates on GitHub by [searching for repositories that start with "soroban-template-"](https://github.com/search?q=%22soroban-template-%22&type=repositories). For this tutorial, we'll use [stellar/soroban-template-astro](https://github.com/stellar/soroban-template-astro). We'll also use a tool called [degit](https://github.com/Rich-Harris/degit) to clone the template without its git history. This will allow us to set it up as our own git project.
```

Comment:
> You can find Soroban templates on GitHub by

> `soroban-template-`


'Soroban template' doesn't feel specific enough and is also oddly over specific.

Soroban is the contract runtime. But a dapp / web app is primarily interacting with the Stellar network, not the contract runtime.

Templates could exist that don't use contracts at all too.

`stellar-template`?

Reply:
Good point, @leighmcculloch. I think `stellar-template-*` is probably a better prefix.

So far, these templates have been focused on interacting with smart contract projects, which is why we started with `soroban-template-*`.

If we want to update this, we will need to update all the [existing templates](https://github.com/search?q=%22soroban-template-%22&type=repositories). These are mostly owned by @ElliotFriend. What do you think, Elliot?

Reply:
Did we consider using repo topics?

Repo topics are convenient because you can actually search and browse them on GitHub, where-as searching for repos with a particular prefix is harder / less accurate and will match on unrelated things.

For example:
https://github.com/topics/stellar

I think we could create a topic `stellar-template`:
https://github.com/topics/stellar-template

Vs searching for repos with a particular prefix, which requires devs to have knowledge of GitHub search queries:
https://github.com/search?q=soroban-template-+in:name&type=repositories

Reply:
Another pro for repo topics is that any repo assigned a topic will have a little blue circle with the topic name in it on it's home page, which when folks click will take them to a browsable list of repos sorted by popularity that are also tagged with the topic. i.e. Repo topics are self discoverable.